packages/clay-multi-step-nav/src/MultiStepNavWithBasicItems.tsx

```diff
@@ -60,6 +60,11 @@ export interface IProps extends React.ComponentProps<typeof ClayMultiStepNav> {
 	 */
 	spritemap?: string;
 
+	/**
+	 * Defines the status of the current step.
+	 */
+	state?: 'error' | 'complete';
```

Comment:
I don't think we should be setting state from the parent component. That should only be set on the item since the error could be on another item than the current one. For instance in the example, when the error is set on the parent component, you can then change the current step and the error moves to whichever step is set as current.

Reply:
This is a high-level component the developers don't have `Item` visibility, although they can define it using the `items` prop but I think it makes sense in a multi step case where the interaction is linear, normally you just move to the next step if you don't get an error I think it makes sense to keep this prop at the component level to define the state of the current step, does it make sense to you?

Reply:
Yeah I think that's fine. I've played with it a little more and the concern I had about switching steps when there's an error and thus having the error switch to a different step can be easily mitigated by disabling switching:
```
const Component = props => {
  const [active, setActive] = useState(1);
  const [error, setError] = useState(false);

  const steps = [
    {
      subTitle: "SubOne",
      title: "One"
    },
    {
      subTitle: "SubTwo",
      title: "Two Long of a title"
    },
    {
      subTitle: "SubThree",
      title: "Three"
    },
    {
      subTitle: "SubFour",
      title: "Four"
    }
  ];

  return (
  <div>
    <ClayMultiStepNavWithBasicItems
      active={active}
      onActiveChange={i => {
        if (!error) {
          setActive(i);
        }
      }}
      spritemap={spritemap}
      state={error ? 'error' : 'complete'}
      steps={steps}
    />
    
    <button className="btn btn-primary" onClick={() => setError(!error)}>Toggle Error</button>
    </div>
  );
};

render(<Component />);
```

If someone wanted to switch steps while there's an error they can always use `<ClayMultiStepNav.Item>` and manage the state at that level. So yes, I see the value in both. Sounds good to me. 👍🏻